### PR TITLE
incr. version range for m2e

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.m2e/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -12,7 +12,7 @@ Require-Bundle: org.eclipse.m2e.core;bundle-version="1.0.0";resolution:=optional
  org.eclipse.xtend.lib,
  org.eclipse.xtext.builder,
  org.eclipse.xtext.ui,
- org.eclipse.m2e.maven.runtime;bundle-version="[1.0.0,1.6.0)";resolution:=optional
+ org.eclipse.m2e.maven.runtime;bundle-version="[1.0.0,1.8.0)";resolution:=optional
```
Comment (kthoms) [inline org.eclipse.xtext.m2e/META-INF/MANIFEST.MF]: Do we need the constraint at all?

Comment (cdietrich) [reply org.eclipse.xtext.m2e/META-INF/MANIFEST.MF]: no idea

Comment (kthoms) [reply org.eclipse.xtext.m2e/META-INF/MANIFEST.MF]: I don't think that we need it. It worked with 1.6 and also 1.8. Likely 1.8+ also. From my point of view you could remove it or make an upper bound to 2.0.0.

Comment (cdietrich) [reply org.eclipse.xtext.m2e/META-INF/MANIFEST.MF]: updated

**Pull request merged**